.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,61 @@
+
+ name: Build Docs
+ on:
+   push:
+     branches:
+     - main
+ jobs:
+   docs:
+     name: Build & Push Docs
+     runs-on: ${{ matrix.os }}
+     timeout-minutes: 90
+     strategy:
+       matrix:
+         os: ['ubuntu-latest']
+         environment-file: [ci/39.yml]
+         experimental: [false]
+     defaults:
+       run:
+         shell: bash -l {0}
+     steps:
+       - uses: actions/checkout@v2
+       - uses: actions/cache@v2
+         env:
+           CACHE_NUMBER: 0
+         with:
+           path: ~/conda_pkgs_dir
+           key: ${{ matrix.os }}-conda-${{ env.CACHE_NUMBER }}-${{ hashFiles(matrix.environment-file) }}
+       - uses: conda-incubator/setup-miniconda@v2
+         with:
+            miniconda-version: 'latest'
+            mamba-version: '*'
+            channels: conda-forge
+            channel-priority: true
+            auto-update-conda: false
+            auto-activate-base: false
+            environment-file: ${{ matrix.environment-file }}
+            activate-environment: test
+            use-only-tar-bz2: true
+       - run: mamba info --all
```

Comment:
these lines wont error, but coda will throw a warning that these commands arent implented. Better to switch them to conda

Reply:
Which commands? The [`info`](https://github.com/pysal/spaghetti/runs/1618153149?check_suite_focus=true#step:5:1) and [`list`](https://github.com/pysal/spaghetti/runs/1618153149?check_suite_focus=true#step:6:1) commands work, but not the [`config`](https://github.com/pysal/spaghetti/runs/1618153149?check_suite_focus=true#step:8:30) command.

Reply:
config is the only one that actually needs changing, but swapped them all because mamba doesnt speed those up and i prefer the info without the mamba graphic at the top